test/types/date.test.ts

```diff
@@ -86,4 +86,26 @@ describe('date', () => {
     expect(await checkInferYear('50')).toBe('1950');
     expect(await checkInferYear('98')).toBe('1998');
   });
+
+
+  it('Changes input before submission', async () => {
+  const { fieldConfigFactory } = TYPE_GENERATORS.date
+    , fieldConfig = fieldConfigFactory.build()
+    , fieldSets = [[fieldConfig]]
+    , onSave = jest.fn()
+    , props = { fieldSets, onSave, required: false }
+    ;
+
+  const tester = await new Tester(FormCard, { props }).mount();
+  //enter normal date
+  tester.changeInput(`input[id="${fieldConfig.field}.month"]`, '01');
+  tester.changeInput(`input[id="${fieldConfig.field}.day"]`, '01');
+  tester.changeInput(`input[id="${fieldConfig.field}.year"]`, '2019');
+  //clear all fields
+  tester.changeInput(`input[id="${fieldConfig.field}.month"]`, '');
+  tester.changeInput(`input[id="${fieldConfig.field}.day"]`, '');
+  tester.changeInput(`input[id="${fieldConfig.field}.year"]`, '');
+  await tester.submit();
+  expect(await !!onSave.mock.calls.length).toBe(true);
```

Comment:
What is this await for???

Reply:
All the other expect() statements had await in them so I assumed it was needed